CMSgov/onemac
# OY2-16029 Submission Form: Submit button updates and Warning Text

Story: https://qmacbis.atlassian.net/browse/OY2-16029
Endpoint: https://d3but3nc1279m0.cloudfront.net/dashboard

### Details
Submission form updates to relocate the submit button to bottom, add some text and change some colors.
Also add a cancel button which functions the same as the back arrow on the PageTitleBar

### Changes
- moved the submit button to the bottom of the form
- added the descriptive text from the story
- re-styled button and text
- added cancel + functionality

### Implementation Notes
Confirmation Dialog now used twice in the page, not sure if this is the best way to incorporate?

### Test Plan
1. Login as a submitter user
2. Go to each form
3. verify that the button is:
  a. moved to bottom
  b. has text above it from the AC
  c. is gray until form is ready, then is green
4. Verify the Cancel functionality
  a. has a confirm modal 
  b. when confirm is canceled, user returned to form with previous entries still there
  c. when confirm is "Leave anyway," user is sent to the previous page.
